### PR TITLE
Add server ID to phone

### PR DESCRIPTION
**Pull request by Bigbearbeasttv**
image

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -257,7 +257,8 @@ local function LoadPhone()
             PhoneData = PhoneData,
             PlayerData = PhoneData.PlayerData,
             PlayerJob = PhoneData.PlayerData.job,
-            applications = Config.PhoneApplications
+            applications = Config.PhoneApplications,
+            serverid = GetPlayerServerId(PlayerId())
```
Comment (jnccloud) [inline client/main.lua:261]: The server ID should already be in PlayerData

File: html/index.html [hunk 2]
```diff
@@ -48,6 +48,7 @@
                     <div id="phone-icons">
                         <i class="fas fa-signal" id="phone-signal" data-toggle="tooltip" data-placement="left" title="QBCore 5G"></i>
                         <i class="fas fa-battery-three-quarters" id="phone-battery"></i>
+                        <i class="fas fa-hashtag" style="padding-right: 10px;" id="phone-serverid" data-toggle="tooltip" data-placement="left" title="Server ID"></i>
```
Comment (jnccloud) [inline html/index.html:51]: Should definitely look at a better way to apply styles here, look at the clock code inside the phone for more details.

File: html/js/app.js [hunk 3]
```diff
@@ -448,6 +448,8 @@ QB.Phone.Functions.LoadPhoneData = function(data) {
     QB.Phone.Functions.LoadMetaData(data.PhoneData.MetaData);
     QB.Phone.Functions.LoadContacts(data.PhoneData.Contacts);
     QB.Phone.Functions.SetupApplications(data);
+    $("#phone-serverid").html("<span style='font-size: 1.3vh; font-weight:bold;'>" + data.serverid + "</span>");
```
Comment (jnccloud) [inline html/js/app.js:451]: Adding attributes like style should be handled in CSS not inline...

File: html/js/app.js [hunk 4]
```diff
@@ -448,6 +448,8 @@ QB.Phone.Functions.LoadPhoneData = function(data) {
...
+    console.log("Phone succesfully loaded!");
```
Comment (jnccloud) [inline html/js/app.js:452]: I think this was removed in the most recent updates? It's a debug print.. doesn't really serve a purpose to anyone.

**Review by jnccloud — CHANGES_REQUESTED** [review]
The player's ID should already be provided in app.js if you access QB.Phone.Data.PlayerData.source in app.js which eliminates the need for adding a new output via any of the .lua scripts. I also recommend using the same styles as the clock because Times New Roman just isn't it in 2021.

**Comment by Bigbearbeasttv** [comment]
Now conflicts with new reskin

**Pull request closed without merging**